scripts/train_with_rllib.py

```diff
@@ -484,7 +484,7 @@ def trainer(
     # Read the run configurations specific to the environment.
     # Note: The run config yaml(s) can be edited at warp_drive/training/run_configs
     # -----------------------------------------------------------------------------
-    config_path = os.path.join(PUBLIC_REPO_DIR, "scripts", "rice_rllib.yaml")
+    config_path = os.getenv("CONFIG_FILE", os.path.join(PUBLIC_REPO_DIR, "scripts", "rice_rllib.yaml"))
```

Comment:
Added an option to overwrite the config file. This may also be useful when trying to run different experiments using this script.